src/community/geopkg/src/test/java/org/geoserver/geopkg/wps/GeoHashCalculatorTest.java

```diff
@@ -18,6 +19,11 @@
     WKTReader wkt = new WKTReader();
     GeoHashCalculator calculator = GeoHashCalculator.DEFAULT;
 
+    @BeforeClass
+    public static void setup() {
+        System.setProperty("org.geotools.referencing.forceXY", "true");
```

Comment:
I would love to find out what code was changing forceXY to false? 

If this is required for the test case use CRS.reset method in both setup and teardown - similar to https://github.com/geotools/geotools/blob/213393c288d52f7e302df1dba6cc3a23f77fe356/modules/plugin/imagepyramid/src/test/java/org/geotools/gce/imagepyramid/AbstractPyramidTest.java#L34-L44

Reply:
Nothing changes it to false. False is the default.

Reply:
But I see what you mean, when multiple tests run in the same JVM, there is a chance a reset in setup might be needed.
It's not actually happening, since integration tests force it to true anyways, the setup was needed anyways:
-  to run the test stand alone
- if surefire happened to run the test as the first one in the build